transformations/numeric_to_word/transformation.py

```diff
@@ -0,0 +1,59 @@
+import random
+import string
+from .numeric2word import recognize_transform
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+class NumericToWord(SentenceOperation):
```

Comment:
For good readability, better to add few lines of comments for this transformation here.

Reply:
done, added docstring from this PR message